assets/dev/js/frontend/utils/url-actions.js

```diff
@@ -29,6 +35,54 @@ export default class extends elementorModules.ViewModule {
 		};
 	}
 
+	/**
+	 * Sanitize Action Settings
+	 *
+	 * This method sanitizes the action settings. If it finds that the settings don't contain a `url` property, or the
+	 * settings contain a URL of an asset (image, video) that does not already exist on the page, it returns false.
+	 *
+	 * If the passed URL does exist on the page, it sanitizes the settings values.
+	 *
+	 * @since 3.5.6
+	 *
+	 * @param {object} settings
+	 * @returns {boolean|object}
+	 */
+	sanitizeActionSettings( settings ) {
```

Comment:
It's always better to keep objects immutable when possible.
In this case, it might be better to create a new settings object, populate it and return the new object without modifying the old one.

Reply:
Done